kubernetes/mlflow-tracking-operator-master/pkg/apis/ai/v1alpha1/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package v1alpha1 contains API Schema definitions for the ai v1alpha1 API group
```

Comment:
Context on what this `v1alpha1` project/package name?

Reply:
This is just a generic version. Since we are making a kubernetes native `Kind` we need to have a version. Did you want to recommend another version name?